addons/mass_mailing/views/snippets/s_showcase.xml

```diff
@@ -6,36 +6,45 @@
         <!-- TODO: (below) issue with height: `fit-content` is not supported, can we calculate it in px in translation ?
         empty div height is 0 unless table has defined height (div.col-1 > div.w-50.h100.border-end)-->
         <div class="container" style="height: fit-content;">
-            <div class="row g-0 s_col_no_resize s_col_no_bgcolor s_nb_column_fixed">
-                <div class="col-lg-6 pb24" style="text-align: right; padding-right: 70px;" align="right">
-                    <div class="mb-2">
-                        <h3 class="d-inline-block">First feature</h3>
-                        <i class="fa fa-2x fa-desktop text-secondary ms-3"/>
+            <div class="row s_col_no_resize s_col_no_bgcolor s_nb_column_fixed">
+                <div class="col-sm text-sm-end">
```

Comment:
In `mass_mailing` with snippet bar and chatter on side I had to change it from `lg` to `sm` at many places. The actual mailing container has less width.